examples/seismic/tutorials/13_LSRTM_acoustic.ipynb

```diff
@@ -0,0 +1,718 @@
+{
```

Comment:
<p>I wouldn't put this one, desn't really mean anything.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/devitocodes/devito/pull/1574/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Reply:
<p>Oh right indeed, missread it. Would it be possible to rename the variables into more meaningful and readable names, such as <code>rtm</code>,  <code>lsrtm</code>,  <code>dm</code>  <span class="ql-cursor">﻿</span></p>

Reply:
Sure! I will do it!